### PR TITLE
Add eks-operator v0.1.0-rc15

### PR DESCRIPTION
**Pull request by rmweir**
**Issue:**
https://github.com/rancher/rancher/issues/26925

### CODE REVIEW

File: charts/rancher-eks-controller/0.1.000/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+name: rancher-eks-controller
+description: A Helm chart for provisioning EKS clusters
+home: https://github.com/rancher/system-charts/charts/eks-controller
```
Comment (cbron) [inline charts/rancher-eks-controller/0.1.000/Chart.yaml]: Believe these should go to code source, not chart source

File: charts/rancher-eks-controller/0.1.000/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+sources:
+  - "https://github.com/rancher/system-charts/charts/eks-controller"
+version: 0.1.000
```
Comment (cbron) [inline charts/rancher-eks-controller/0.1.000/Chart.yaml]: Doesn't need to be triple zeroes, you can just do 0.1.0

File: charts/rancher-eks-controller/0.1.000/templates/eksclusterconfig.yaml [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+            nodeGroups:
+              items:
+                properties:
+                  desiredSize:
+                    nullable: true
+                    type: integer
+                  diskSize:
+                    nullable: true
+                    type: integer
+                  ec2SshKey:
```
Comment (cbron) [inline charts/rancher-eks-controller/0.1.000/templates/eksclusterconfig.yaml]: Is the ssh key still a thing ?

Comment (rmweir) [reply charts/rancher-eks-controller/0.1.000/templates/eksclusterconfig.yaml]: yes it is, it still uses ec2 node which you can ssh into

File: charts/rancher-eks-controller/0.1.000/Chart.yaml (hunk 2 above)
Comment (rmweir) [reply charts/rancher-eks-controller/0.1.000/Chart.yaml]: @deniseschannon  requested this

**Review by cbron — DISMISSED** [review]
lgtm

**Review by brendarearden — DISMISSED** [review]
L🔥TM

**Comment by cbron** [comment]
Looks fine other than triple zeroes.

**Comment by cbron** [comment]
And doing custom Role, not binding to cluster-admin ClusterRole.

File: charts/rancher-eks-operator/0.1.0/templates/clusterrolebinding.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (cbron) [inline charts/rancher-eks-operator/0.1.0/templates/clusterrolebinding.yaml:1]: Can this be roleBinding only to cattle-system NS ? Or do we need access to other NS’s

Comment (rmweir) [reply charts/rancher-eks-operator/0.1.0/templates/clusterrolebinding.yaml:1]: It needs to access cattle-global-data. That's where the secrets and EKSClusterConfigs are.

Comment (cbron) [reply charts/rancher-eks-operator/0.1.0/templates/clusterrolebinding.yaml:1]: Believe we need to make 2 role bindings then for both namespaces.

File: charts/rancher-eks-operator/0.1.0/templates/serviceaccount.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: cattle-system
+  name: eks-config
```
Comment (cbron) [inline charts/rancher-eks-operator/0.1.0/templates/serviceaccount.yaml]: nit: name of eks-operator

**Review by cbron — DISMISSED** [review]

**Review by cbron — DISMISSED** [review]

**Review by brendarearden — APPROVED** [review]
LGTM

**Review by cbron — APPROVED** [review]

**Review by aiyengar2 — APPROVED** [review]

**Pull request merged**